Presentation/MasterDetailSelection.swift

```diff
@@ -34,8 +34,9 @@ public final class MasterDetailSelection<Elements: BidirectionalCollection>: Sig
 
         let subBag = DisposeBag()
         bag += subBag
-        bag += isCollapsed.atOnce().onValue { isCollapsed in
+        bag += isCollapsed.atOnce().onValue { [weak self] isCollapsed in
```

Comment:
Should we update to use `onValueDisposePrevious` here instead?

Reply:
Fixed, can you approve again?